packages/core/android/detekt.yml

```diff
@@ -329,7 +329,7 @@ naming:
     excludeClassPattern: '$^'
     ignoreOverridden: true
   InvalidPackageDeclaration:
-    active: true
+    active: false
```

Comment:
Here I'm disabling this rule to match a bit more the "traditional" RN architecture and to avoid having too many nested folders (in `main`, `newarch` and `oldarch`). 
As we have to also make sure the path are correct in the sourceSets, I believe this reduces a bit the chances of making a mistake there.

Let me know if you think we should instead keep this rule.